.travis.yml

```diff
@@ -50,7 +50,7 @@ before_script:
     - pep8 --config ./.pep8.rc --exclude=client .
 
     # Prepare the build and everything else
-    - ./.travis/before_script.sh
+    - mozdownload --type daily --branch $TRAVIS_BRANCH --locale $LOCALE
```

Comment:
As I have said on another PR for this bug, we cannot use the $TRAVIS_BRANCH environment variable. Instead we will have to define our own one.